src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,118 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++;
+ *   case 2: // violation, case 1 contains code but lacks break, return, throw or continue statement
```

Comment:
`violation, case 1 contains code but lacks break, return, throw or continue statement`
1. continue cannot be in switch, please correct description as well.
2. `violation, case 1 ` -> `violation, previous case ...`

Reply:
In my code, I have a `while loop`, so `continue` is possible. I leave the `while loop` out to focus on `switch`.
```
while (i >= 0) {
    switch(i) {
        case 1:
            i++;
            continue;
        ...
    }
}
```

Reply:
About your review number 2, I'll fix that

Reply:
Then you example should have loop outside of it, because continue wont work without a loop and it is confusing

Reply:
Ok I'll fix that

Reply:
fixed